ts/components/core/IOBadge.tsx

```diff
@@ -1,19 +1,19 @@
+import { Badge } from "native-base";
 import React from "react";
 import { ColorValue, StyleSheet } from "react-native";
-import { Badge } from "native-base";
-import { IOColors } from "./variables/IOColors";
 import { LabelSmall } from "./typography/LabelSmall";
+import { IOColors, IOColorType } from "./variables/IOColors";
 
 type IOBadgeCommonProps = {
   text: string;
   small?: boolean;
-  labelColor?: "white" | "blue";
+  labelColor?: Extract<IOColorType, "bluegreyDark" | "blue" | "white" | "red">;
 };
 
 const commonBadgeStyles = StyleSheet.create({
   badge: {
-    paddingLeft: 15,
-    paddingRight: 15
+    paddingLeft: 8,
+    paddingRight: 8
```

Comment:
@fabriziofff did you change the padding also for non-small badges on purpose?

Reply:
@emiliopavia yes, I followed the specification provided in https://www.figma.com/file/GBRk8f5WGyaX29lahpT5fM/App-IO---Design-System?node-id=154%3A342. The non-small badges are not provided by design, but I changed the padding in order to have the same (similar) style.

Reply:
Maybe @thisisjp could help us to undestand if the same padding is ok?
![immagine](https://user-images.githubusercontent.com/26501317/172149664-7120ac64-737d-4e6e-ac55-c574a26b036c.png)

Reply:
Figma is always the source of truth :) We'll review all the components in any case sooner or later...

(btw: where did you see this small badge design? it's not part of our library 👀 )

cc @dmnplb

Reply:
I've used https://www.figma.com/file/GBRk8f5WGyaX29lahpT5fM/App-IO---Design-System?node-id=154%3A342 as reference 😄 Did I used the wrong file?

Reply:
@emiliopavia we agreed with @thisisjp to keep the same padding. In a next iteration all the "non-small" badges should be removed from the codebase.